android/build.gradle

```diff
@@ -22,7 +22,7 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 32
```

Comment:
Isn't android 31 the latest stable sdk version? or am I missing something?

Reply:
Hello, 

Android 12 (Sdk 32) has been recently quit from the "Preview" mode and became the new default, also I can confirm this by the increasing percentage of Android 12 phones, using my apps on the play store. So I thought it would be better to increase the targetSdkVersion and compileSdkVersion to highest.

(Android studio will soon start to show deprecation warnings for Sdk 31 soon anyway)

-P.S android studio suggests installing SDK 32 on new ide installations so it comes with SDK 32 by default

Reply:
Have you tested it in a android 32 emulator?
Can you please downgrade it to 31.
The latest flutter (2.10.3) also has made android SDK versions to 31 and I don't think many people are using beyond that.

The reason is, if the user app is supporting 31 and the plugin requires 32 it will throw a warning
![image](https://user-images.githubusercontent.com/53572638/158718096-69c766bf-d4c4-449c-ac1d-da63a25ceaeb.png)

not that its wrong, but lets support SDK 32 when required.

Reply:
Hello,

I have tested it on the Android 32 emulator, and no problems.

I think we are in a lts/stable version shifting phase 
![image](https://user-images.githubusercontent.com/12881878/158719131-ba3d601c-e3ad-49a7-8a5b-9de615622935.png)

But I will downgrade it to 31, until the mainstream becomes 32 for the masses